brick/__brick__/{{project_name.snakeCase()}}/analysis_options.yaml

```diff
@@ -1,4 +1,4 @@
-include: package:very_good_analysis/analysis_options.3.0.2.yaml
+include: package:very_good_analysis/analysis_options.yaml
```

Comment:
we shouldn't modify the contents of the brick directly since it is generated. Instead, we should update the contents of the `src` directory

Reply:
See #177 for an example and sorry for the confusion!